hles/transformation/src/test/scala/org/broadinstitute/monster/dap/DogTransformationsSpec.scala

```diff
@@ -11,20 +11,22 @@ class DogTransformationsSpec extends AnyFlatSpec with Matchers with OptionValues
 
   it should "map required fields" in {
```

Comment:
Would be nice to have a test case that 1) verifies `None` is returned and 2) verifies the counter is incremented. Verifying logging is nice too; not sure how hard it would be to do though.